contracts/Safe.sol

```diff
@@ -309,11 +309,11 @@ contract Safe is
      */
     function checkNSignatures(address executor, bytes32 dataHash, bytes memory signatures, uint256 requiredSignatures) public view {
         // Check that the provided signature data is not too short
-        require(signatures.length >= requiredSignatures.mul(65), "GS020");
+        if (signatures.length < requiredSignatures.mul(65)) revertWithError("GS020");
         // There cannot be an owner with address 0.
         address lastOwner = address(0);
         address currentOwner;
-        uint8 v;
+        uint256 v;
```

Comment:
This is the only `uint8 v` to `uint256 v` change I would make (including dependencies, as it should be the only one that affects codesize.

Reply:
If we don't make the change in the `SignatureDecoder:signatureSplit(...)`, then in line [321](https://github.com/safe-global/safe-contracts/pull/726/files/85f043f26b63ce4e5daf4723d019466d1922d9b0#diff-587b494ea631bb6b7adf4fc3e1a2e6a277a385ff16e1163b26e39de24e9483deL321) we would need to use another temp variable, which defeats the purpose.

Reply:
> If we don't make the change in the `SignatureDecoder:signatureSplit(...)`

Agreed. This is what I meant with:

> including dependencies

Reply:
Sorry - I think I didn't explain myself well.

What I meant was this was the place where we should consider using `uint256` instead of `uint8` for `v`, and that we should update any other code dependencies that stem from that change (including changes to `SignatureDecoder:signatureSplit`).

I guess I was just surprised that we needed to change the Certora spec, but if it is really needed, then we should. The reason I was surprised is that it looks like unrelated "helper" methods related to signatures that could have been kept as `uint8`.

Reply:
Well, implicit conversion did its work. So, all good 👍🏾